File: Part4/Python/Function_IncorrectCapitalization.py
def main():
    print( "Type 1 and press enter." )
    user_input = input()
    
    # Function name has wrong capitalization
    entered_integer = Int( user_input )
    
    print( f"The user entered the integer { entered_integer }" )

if __name__ == "__main__":
    main()

File: Part4/C#/Function_IncorrectCapitalization.cs
using System;

class Program
{
    static void Main()
    {
        string userInput;
        
        Console.WriteLine("Type 1 and press enter.");
        userInput = Console.ReadLine();
        
        // Function name has wrong capitalization
        int enteredInteger = int.parse(userInput);
        
        Console.WriteLine("The user entered the integer {0}", enteredInteger);
    }
}
